src/test/java/htsjdk/tribble/bed/BEDCodecTest.java

```diff
@@ -226,4 +226,15 @@ private void createIndex(File testFile, File idxFile) throws IOException {
     public void testGetTabixFormat() {
         Assert.assertEquals(new BEDCodec().getTabixFormat(), TabixFormat.BED);
     }
+
```

Comment:
I will add it tomorrow, I don't have now a command line available to create the test file.